frontend/src/utils/select-utils.js

```diff
@@ -0,0 +1,156 @@
+  { id: 39, value: "South Carolina" },
+  { id: 40, value: "South Dakota" },
+  { id: 41, value: "Tennessee" },
+  { id: 42, value: "Texas" },
+  { id: 43, value: "Utah" },
+  { id: 44, value: "Vermont" },
+  { id: 45, value: "Virginia" },
+  { id: 46, value: "Washington" },
+  { id: 47, value: "West Virginia" },
+  { id: 48, value: "Wisconsin" },
+  { id: 49, value: "Wyoming" },
+];
+
+export const utilityOptions = [
+  { id: 0, value: "Water" },
+  { id: 1, value: "Electricity" },
+  { id: 2, value: "Gas" },
+  { id: 3, value: "Sewer" },
+  { id: 4, value: "Internet" },
+  { id: 5, value: "Telephone" },
+  { id: 6, value: "TV" },
+  { id: 7, value: "Other (Type in)" },
+];
+
+export const insuranceOptions = [
+  { id: 0, value: "Health Insurance" },
+  { id: 1, value: "Life Insurance" },
+  { id: 2, value: "Dental Insurance" },
+  { id: 3, value: "Car Insurance" },
+  { id: 4, value: "Homeowners/Renters Insurance" },
+  { id: 5, value: "Life Insurance" },
+  { id: 6, value: "Flood Insurance" },
+  { id: 7, value: "Pet Insurance" },
+  { id: 8, value: "Disability Insurance" },
+  { id: 9, value: "Umbrella Insurance" },
+  { id: 10, value: "Other (Type in)" },
+];
+
+export const housingOptions = [
+  { id: 0, value: "Rent" },
+  { id: 1, value: "Mortgage" },
+  { id: 2, value: "Other (Type in)" },
+];
+
+export const billOptions = [
+  { id: 0, value: "Rent" },
+  { id: 1, value: "Mortgage" },
+  { id: 2, value: "Lawn Service" },
+  { id: 3, value: "Water" },
+  { id: 4, value: "Electricity" },
+  { id: 5, value: "Gas" },
+  { id: 6, value: "Sewer" },
+  { id: 7, value: "Internet" },
+  { id: 8, value: "Telephone" },
+  { id: 9, value: "TV" },
+  { id: 10, value: "Other (Type in)" },
+];
+
+export const shoppingOptions = [
+  { id: 0, value: "Groceries" },
+  { id: 1, value: "Clothing/Shoes" },
+  { id: 2, value: "Other (Type in)" },
+];
+
+export const leisureOptions = [
+  { id: 0, value: "Concert" },
+  { id: 1, value: "Dining" },
+  { id: 2, value: "Vacation" },
+  { id: 3, value: "Sports" },
+  { id: 4, value: "Other (Type in)" },
+];
+
+export const transportationOptions = [
+  { id: 0, value: "Car" },
+  { id: 1, value: "Bus" },
+  { id: 2, value: "Train" },
+  { id: 3, value: "Flight" },
+];
+
+export const subscriptionOptions = [
+  { id: 0, value: "Music Subscription (Spotify, Apple Music)" },
+  { id: 1, value: "TV/Movie Subscription (Netflix, Hulu, Disney+)" },
+  { id: 2, value: "Gym Membership" },
+  { id: 3, value: "Magazine Subscription" },
+  { id: 4, value: "Other (Type in)" },
+];
+
+export const loandebtOptions = [
+  { id: 0, value: "Credit Card" },
+  { id: 1, value: "School/Student" },
+  { id: 2, value: "Medical" },
+  { id: 4, value: "Other (Type in)" },
+];
+
+export const childEducationOptions = [
+  { id: 0, value: "College" },
+  { id: 1, value: "Going to College" },
+  { id: 2, value: "Other" },
+];
+
+let maxOffset = 120;
+let thisYear = new Date().getFullYear();
+export let birthYearOptions = [];
```

Comment:
i think this one should be a function instead, cuz its got more going on and the value for maxOffset changes so u can pass that in as a param when u use the function